src/lv_conf.h

```diff
@@ -230,7 +230,7 @@
  *-----------*/
 
 /*Enable the log module*/
-#define LV_USE_LOG 0
+#define LV_USE_LOG 1
```

Comment:
I think it's better to keep this to what it was, same as the options below.

Reply:
I'll set LOG and LEVEL back. I'd like to leave PRINTF at 1 - this will save the next person 15 minutes of headscratching :)